models/articles-models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+function fetchArticleById(article_id) {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "invalid article ID" });
+      }
+      return rows[0];
```

Comment:
Nice manual handling of this `404` error! 

What would happen if the `article_id` was a word instead of a number, how would your API need to handle that ?